components/smalltalk/squeak4/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Package Makefile for the "Classic VM" Squeak Smalltalk V4 system
+# See http://squeak.org and http://squeakvm.org
+#
+
+# squeak4 uses cmake and a custom configure script (not GNU autoconf)
+BUILD_STYLE=		configure
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		squeak4
+COMPONENT_VERSION=	4.16.7
+COMPONENT_SUMMARY=	The Squeak V4 Smalltalk System
+COMPONENT_PROJECT_URL=	http://www.squeak.org
+COMPONENT_FMRI=		runtime/squeak4
+COMPONENT_CLASSIFICATION=	Development/Other Languages
+SVN_REPO=		http://squeakvm.org/svn/squeak/trunk/
+SVN_REV=		3775
+SVN_HASH=  sha256:497a8cc645a75fc049f9467dab6fff2bd06726e09c39d1a4d1a42164f820a102
+SQUEAK_REV = $(COMPONENT_VERSION)-$(SVN_REV)
+SQUEAK_VERSION = squeak-$(SQUEAK_REV)
+COMPONENT_LICENSE=	MIT
+COMPONENT_LICENSE_FILE=	usr/share/doc/$(SQUEAK_VERSION)/LICENSE
+
```

Comment:
We don't use the target path for the license file. Typically we copy the appropriate license file into the package folder (that is, make it an explicit part of your commit) and just name it here, like
COMPONENT_LICENSE= LICENSE
You can even omit the copying if the file is in the top level folder of the $COMPONENT_ARCHIVE.

Reply:
ok thanks, I'll give this a try

Reply:
Additionally you'll need to add
TEST_TARGET= $(NO_TESTS)
before you include common.mk.